app/views/pages/home.html.erb

```diff
@@ -1,2 +1,30 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+
+
+<div class="content">
+  <div class="content__container">
+    <p class="content__container__text">
+      MAKE YOU
+    </p>
+
```

Comment:
Let's keep consistent with Le Wagon's code style. We've never used double-underscores before. I don't really understand it... 🤔 💭